tests/integration/cmor/_fixes/cesm/test_cesm2.py

```diff
@@ -255,6 +275,38 @@ def test_only_longitude(monkeypatch):
                                [[-90.0, 90.0], [90.0, 270.0]])
 
 
+# Test AllVars._fix_time
+
+
+def test_fix_time_mon(cube_1d_time):
+    """Test `_fix_time``."""
+    fix = get_allvars_fix('Amon', 'mon', 'tas')
+    fix._fix_time(cube_1d_time)
+    time_coord = cube_1d_time.coord('time')
+    np.testing.assert_array_equal(time_coord.points, [1, 3, 5])
+    np.testing.assert_array_equal(time_coord.bounds, [[0, 2], [2, 4], [4, 6]])
+
+
+def test_fix_time_mon_point(cube_1d_time):
+    """Test `_fix_time``."""
+    cube_1d_time.add_cell_method(CellMethod('point', 'time'))
+    fix = get_allvars_fix('Amon', 'mon', 'tas')
+    fix._fix_time(cube_1d_time)
+    time_coord = cube_1d_time.coord('time')
+    np.testing.assert_array_equal(time_coord.points, [2, 4, 6])
+    np.testing.assert_array_equal(time_coord.bounds, [[0, 2], [2, 4], [4, 6]])
+
+
+def test_fix_time_day(monkeypatch, cube_1d_time):
+    """Test `_fix_time``."""
+    fix = get_allvars_fix('Amon', 'mon', 'tas')
+    monkeypatch.setitem(fix.extra_facets, 'frequency', 'day')
+    fix._fix_time(cube_1d_time)
+    time_coord = cube_1d_time.coord('time')
+    np.testing.assert_array_equal(time_coord.points, [2, 4, 6])
+    np.testing.assert_array_equal(time_coord.bounds, [[0, 2], [2, 4], [4, 6]])
```

Comment:
but where was the fix applied? These are identical to the input in `cube_1d_time`? Or am I blind :eyeglasses:

Reply:
Yes, the fix does indeed nothing here, since it looks like daily data. As agreed with @zklaus, the fix is only supposed to change monthly data, all other data is unchanged.

Reply:
great, cheers, Manu!